src/libraries/System.Data.Common/src/System/Xml/XPathNodePointer.cs

```diff
@@ -376,7 +374,7 @@ internal string InnerText
                 }
                 else
                 {
-                    DataRow row = Row;
+                    DataRow row = Row!;
```

Comment:
Another possible NRE from using the `Row` property

Reply:
See https://github.com/dotnet/runtime/pull/41541#discussion_r479606473